dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_countersigned_agreement_document_path
+
+
+def upload_file(bucket, dry_run, file_path, framework_slug):
+    supplier_id = get_supplier_id_from_framework_file_path(file_path)
+    upload_path = get_countersigned_agreement_document_path(framework_slug, supplier_id)
+    if not dry_run:
+        with open(file_path) as file_contents:
+            bucket.save(upload_path, file_contents, acl='public-read', move_prefix=None)
+        print(supplier_id)
+    else:
+        print("[Dry-run] UPLOAD: '{}' to '{}'".format(file_path, upload_path))
+
+
+def get_bucket_name(stage, document_type):
+    bucket_name = 'digitalmarketplace-{0}-{1}-{1}'.format(document_type, stage)
+    print("BUCKET: {}".format(bucket_name))
+    return bucket_name
+
+
+def get_all_files_of_type(local_directory, file_type):
+    for root, subfolder, files in os.walk(local_directory):
+        for filename in files:
+            if filename.endswith(file_type):
+                yield os.path.join(root, filename)
+
+
+def get_supplier_id_from_framework_file_path(path):
+    # Filenames from CCS are in the format:
```

Comment:
That was too specific for a regex.  Thought the formats they were using might be useful to know in the future if things changed.  Can take the comment out though.  I think the regex is self explanatory.